### PR TITLE
wavesj improvements

### PR DESCRIPTION
**Pull request by DrBlast**

### CODE REVIEW

File: src/main/java/com/wavesplatform/wavesj/Transactions.java [hunk 1]
```diff
@@ -16,6 +16,11 @@ public static IssueTransactionV2 makeIssueTx(PrivateKeyAccount sender, byte chai
         return new IssueTransactionV2(sender, chainId, name, description, quantity, decimals, reissuable, script, fee, timestamp);
     }
 
+    public static IssueTransactionV1 makeIssueV1Tx(PrivateKeyAccount sender, byte chainId, String name, String description,
```
Comment (petermz) [inline src/main/java/com/wavesplatform/wavesj/Transactions.java]: Should be removed

File: src/main/java/com/wavesplatform/wavesj/Node.java [hunk 2]
```diff
@@ -105,6 +123,24 @@ public long getBalance(String address, String assetId) throws IOException {
...
+        return parse(r, ASSET_DISTRIBUTION);
+    }
+
+    public Map<String, Long> getAssetDistributionByHeight(String assetId, Integer height) throws IOException {
+        String path = String.format("/assets/%s/distribution/%d", assetId, height);
+        HttpResponse r = exec(request(path));
+        return parse(r, ASSET_DISTRIBUTION);
+    }
+
+    public List<AssetBalance> getAssetsBalance(String address) throws IOException {
```
Comment (petermz) [inline src/main/java/com/wavesplatform/wavesj/Node.java]: What about Map<String, Long> ? Is it any more convenient?

Comment (DrBlast) [reply src/main/java/com/wavesplatform/wavesj/Node.java]: This endpoint not returns a Map<String, Long> It returns json with asset balance and  asset details, with inner object issue tx info.

File: src/main/java/com/wavesplatform/wavesj/Transactions.java (hunk 1 above)
Comment (DrBlast) [reply src/main/java/com/wavesplatform/wavesj/Transactions.java]: ok

**Pull request merged**